blueprint/index.md

```diff
@@ -240,156 +241,151 @@ The Microsoft Teams video session URL is sent as an SMS to the customer from Gen
+   ::: 
 
-  ![Map First Data Action Input Variable](images/mapFirstDataActionDefineUserInputVariable.png "Map First Data Action Input Variable")
+      ![Map First Data Action Input Variable](images/mapFirstDataActionDefineUserInputVariable.png "Map First Data Action Input Variable")
 
-10. Define static values for the remaining input variables.
+9. Define the static values for the input variables, `startTime`, `endTime`, and `timeZone`. 
 
-  :::primary
-  **Note:** Using the values in this example will create ISO-8601 formatted timestamps.  The startTime and endTime parameters can be the same and it can occur in the past.  If you would like to define your own static timestamps for these variables, you are welcome to do so.
-  :::
+   :::primary
+   **Note:** You can enter the same value for `startTime` and `endTime` parameters and the timestamp can be in the past. Also, define your own timestamp format. The example shows the value to create the ISO-8601 format timestamp. 
+   :::
 
-  ![Define Static Input Variables](images/mapFirstDataActionDefineInputVariables.png "Define Static Input Variables")
+   ![Define Static Input Variables](images/mapFirstDataActionDefineInputVariables.png "Define Static Input Variables")
 
-11. Expand the second data action.
+10. Expand the second data action that sends the SMS.
 
-   ![Expand the second data action.](images/expandSecondDataAction.png "Expand the second data action.")
+    ![Expand the second data action.](images/expandSecondDataAction.png "Expand the second data action.")
 
-12. From the **Category** drop menu, select the category of your "Send SMS" data action.  From the **Data Action** drop menu, select your "Send SMS" data action.
+11. From the Category list, select the category of the Send SMS data action.  From the Data Action list, select the Send SMS data action.
 
- ![Map Second Data Action](images/mapSecondDataAction.png "Map Second Data Action")
+    ![Map Second Data Action](images/mapSecondDataAction.png "Map Second Data Action")
 
-13. Expand the input variables for the Second Data Action
+12. Expand the input variables for the second data action.
 
-14. Type one of your purchased SMS Numbers from your Genesys Cloud organization in the **fromAddress** input variable.
+13. In the **fromAddress** input variable, type one of your SMS numbers bought from the Genesys Cloud organization.
 
- :::primary
- **Note:** See additional resource below for detailed steps for purchasing an SMS number.  Phone must be typed in +11234567890 format.
- :::
+      :::primary
+      **Note:** See [Purchase SMS long code numbers](https://help.mypurecloud.com/articles/purchase-sms-long-code-numbers/ "Opens the Purchase SMS long code numbers article") in the Genesys Cloud Resource Center for detailed steps about purchasing an SMS number. You must enter the SMS number in +11234567890 format.
+      :::
```

Comment:
For cross references, use the phrasing..."For more information about..., see LINK."